webhook/resourcesemantics/validation/validation_admit.go

```diff
@@ -180,7 +180,7 @@ func validate(ctx context.Context, resource resourcesemantics.GenericCRD, req *a
 	}
 
 	if result := resource.Validate(ctx); result != nil {
-		logger.Errorw("Failed the resource specific validation", zap.Error(err))
+		logger.Infow("Failed the resource specific validation", zap.Error(result))
```

Comment:
I am wondering if we actually need to keep that logging statement given the comment [here](https://github.com/knative/pkg/pull/1172/files#r402569907).
cc @dprotaso

Reply:
I'd be open to that change in a new PR